tests/test_edgeql_syntax.py

```diff
@@ -383,12 +390,12 @@ def test_edgeql_syntax_constants_34(self):
 
     def test_edgeql_syntax_constants_35(self):
         r"""
-        SELECT r"\n\w\d";
```

Comment:
Double quotes are valid for string literal, please keep the test as is.

Reply:
Fixed